ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/processes/panel/ProcessesPanelPresenter.java

```diff
@@ -674,12 +664,7 @@ public void onStopCommandProcess(ProcessTreeNode node) {
 
     @Override
     public void onCloseCommandOutputClick(final ProcessTreeNode node) {
-        closeCommandOutput(node, new SubPanel.RemoveCallback() {
-            @Override
-            public void remove() {
-                view.hideProcessOutput(node.getId());
-            }
-        });
+        closeCommandOutput(node, () -> {});
```

Comment:
I think no, because it's can do logic more difficult because we should add null checking in the few methods.